x/stake/keeper/distribute_sidechain.go

```diff
@@ -339,3 +377,68 @@ func removeValidatorsAndDelegationsAtHeight(height int64, k Keeper, ctx sdk.Cont
 	}
 	k.RemoveValidatorsByHeight(ctx, height)
 }
+
+func crossDistributeReward(k Keeper, ctx sdk.Context, rewardCAoB sdk.AccAddress, delAddr sdk.AccAddress, amount int64) (sdk.Events, error) {
+	relayFeeCalc := fees.GetCalculator(types.CrossDistributeRewardRelayFee)
+	if relayFeeCalc == nil {
+		return sdk.Events{}, fmt.Errorf("no fee calculator of transferOutRewards")
+	}
+	relayFee := relayFeeCalc(nil)
+	bscRelayFee := bsc.ConvertBCAmountToBSCAmount(relayFee.Tokens.AmountOf(k.BondDenom(ctx)))
+
+	bscTransferAmount := new(big.Int).Sub(bsc.ConvertBCAmountToBSCAmount(amount), bscRelayFee)
+	delBscAddrAcc := types.GetStakeCAoB(delAddr.Bytes(), types.DelegateCAoBSalt)
```

Comment:
To avoind funds stuck in `delAddr`, we need check the balance of `delAddr` and transfer all of them, don't we?

Reply:
Yes. That's exactly what I do. The balance is passed in as `amount`